lib/api/src/modules/url.ts

```diff
@@ -76,7 +76,7 @@ const initialUrlSupport = ({ navigate, location, path }: Module) => {
     setTimeout(() => navigate(`/story/*`, { replace: true }), 1);
   } else if (Object.keys(query).length > 1) {
     // remove other queries
-    setTimeout(() => navigate(`/${queryPath}`, { replace: true }), 1);
```

Comment:
What's up with this change?

Reply:
Noticed while testing the copy feature, a `/` would get prepended every time..

(you can actually see this in the video)

This fixes it